tests/unit_tests/Commands/Remote/DrushCommandTest.php

```diff
@@ -23,8 +23,14 @@ public function setUp()
             ->setMethods([
                 'prepareEnvironment',
                 'executeCommand',
+                'log',
```

Comment:
Sorry nobody told you before :( We're really not using anything under the `unit_tests` folder afaik

Reply:
correct. They're woefully out of date.